src/bridge/IrcHandler.ts

```diff
@@ -448,6 +448,55 @@ export class IrcHandler {
         }
     }
 
+    /**
+     * If configured, check to see if the all Matrix users in a given room are
+     * joined to a channel. If they are not, drop the message.
+     * @param req The IRC request
+     * @param server The IRC server.
+     */
+    private async requiresAllMatrixUsersJoined(server: IrcServer, channel: string, roomId: string): Promise<boolean> {
```

Comment:
Why sometimes `requiresAllMatrixUsersJoined` and other times `shouldRequireMatrixUserJoined`?
If they are on different classes but behave the same I'd call them the same.

Reply:
Renamed to be the same.